src/main/include/Utils.h

```diff
@@ -0,0 +1,61 @@
+
+#pragma once
+
+#include <iostream>
+
```

Comment:
This file will conflict with master after #34. Please be careful as to not overwrite that change